Automattic/delphin
# Update the title on the setup completed step to include an illustration

Addresses the "Setup Complete" part of #764. The domain setup step should match the screenshot in #764:

<img width="627" alt="screen shot 2016-10-30 at 15 49 13" src="https://cloud.githubusercontent.com/assets/275961/19837345/779555b8-9eb8-11e6-9825-1d3d062764bc.png">

I decided to split #764 into smaller PRs to make reviewing easier.
#### Testing instructions
1. Run `git checkout update/setup-text` and start your server, or open a [live branch](https://delphin.live/?branch=update/setup-text)
2. Open the [`My Domains` page](http://delphin.localhost:1337/)
3. Go through the flow and check that the last step matches the screnshot in #764
#### Reviews

@Automattic/sdev-feed
